e2e/suite1/specs/inspect-drawer.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Inspect drawer tests',
+  itName: 'Testes various Inpect Drawer scenarios',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: true,
+  skipScenario: false,
+  scenario: () => {
+    const viewPortWidth = e2e.config().viewportWidth;
+    // @todo remove `@ts-ignore` when possible
+    // @ts-ignore
+    e2e.getScenarioContext().then(({ lastAddedDashboardUid }) => {
+      e2e.flows.openDashboard(lastAddedDashboardUid);
+    });
+    e2e.pages.Dashboard.Toolbar.toolbarItems('Add panel').click();
+    e2e.pages.AddDashboard.addNewPanel().click();
+
+    e2e.components.DataSource.TestData.QueryTab.scenarioSelect().select('CSV Metric Values');
+
+    e2e.components.BackButton.backArrow().click();
+
```

Comment:
Not sure we need this step in this case, now with devenv data sources and dashboards you could open one of the devenv dashboards, for example: http://localhost:3000/d/5SdHCadmz/panel-tests-graph?orgId=1  has some graphs that use csv data already.

Reply:
Sure I'll look into that, is the uid in the gdev dashboard always the same?

Reply:
Fixed :)